src/app/containers/Ad/MPU.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_TRPL,
+  GEL_SPACING_QUAD,
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+} from '@bbc/gel-foundations/spacings';
+import styled from 'styled-components';
+import BaseAdContainer from '.';
+
+const AdContainer = styled(BaseAdContainer)`
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    /* To remove GEL Margins */
+    margin: ${GEL_SPACING_QUAD} -${GEL_MARGIN_BELOW_400PX} 0;
+    padding: 0 ${GEL_MARGIN_BELOW_400PX};
+  }
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    margin: ${GEL_SPACING_QUAD} -${GEL_MARGIN_ABOVE_400PX} 0;
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin: ${GEL_SPACING_TRPL} -${GEL_MARGIN_ABOVE_400PX} 0;
+  }
+  /* MPU gets misaligned with the original padding at smaller breakpoints */
```

Comment:
what is meant by the 'original padding' here? Again, it would be good for a prop to be used to disable this undesired behaviour rather than modifying the css at this layer.

Reply:
I'm not sure - I found the commit where this comment was added (as a result of a UX review) - https://github.com/bbc/simorgh/commit/68424d58cf789fa08bbd30b9445d1f16dbceb101